deploy/backup-s3.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: cluster1-s3-credentials
+  finalizers:
```

Comment:
it is the secret why did you add it to the secret

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/158/commits/c5333d99b8176fd5a69c8fdb6b927ae0f585f5b3